ci/licenses.sh

```diff
@@ -21,7 +21,7 @@ unset CDPATH
 function follow_links() (
   cd -P "$(dirname -- "$1")"
   file="$PWD/$(basename -- "$1")"
-  while [[ -h "$file" ]]; do
+  while [[ -L "$file" ]]; do
```

Comment:
I remember seeing a change in this flag between `-h` and `-L` in this or another PR, but I don't remember which is the new or old value, and which is correct. I think we discussed keeping the old value?